Corona-Warn-App/build.gradle

```diff
@@ -318,6 +318,7 @@ dependencies {
     // Required for rdgca.verifier.app:dgc-certlogic-android-light API 26 Time API
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
 
+    implementation 'org.bouncycastle:bcpkix-jdk15on:1.56'
```

Comment:
Wondering why this specific version :thinking: 
https://github.com/corona-warn-app/cwa-app-android/pull/3773#discussion_r674680548

Reply:
@jurajkusnier  do you know?

Reply:
hm.. let's try the latest version `1.69`
it could be just mistake

Reply:
updated